tutorials/quantum_simulation/Building 1D Rydberg Crystals.ipynb

```diff
@@ -139,16 +139,16 @@
      "name": "stdout",
```

Comment:
<p>Your computer seems to be struggling x)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pasqal-io/Pulser/pull/656/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='38'/>

Reply:
<p>I reran all the notebooks + I have long simulations going on ^^'</p>